### ISSUE TITLE
 java.lang.VerifyError crash on Android app startup when using Android plugin in 3.2.4-rc2

### ISSUE THREAD

**Issue by msoechting** [GameAnalytics/GA-SDK-GODOT#11]
With Godot 3.2.4 apparently the way the Godot activity is retrieved has changed, leading to a crash on startup when using the GameAnalytics SDK in an Android build:

```
02-13 23:07:07.393  4669  4669 E AndroidRuntime: FATAL EXCEPTION: main
02-13 23:07:07.393  4669  4669 E AndroidRuntime: Process: com.gameanalytics.godottest, PID: 4669
02-13 23:07:07.393  4669  4669 E AndroidRuntime: java.lang.VerifyError: Verifier rejected class com.gameanalytics.godotgameanalytics.GodotGameAnalytics: void com.gameanalytics.godotgameanalytics.GodotGameAnalytics.init(java.lang.String, java.lang.String) failed to verify: void com.gameanalytics.godotgameanalytics.GodotGameAnalytics.init(java.lang.String, java.lang.String): [0x7] register v0 has type Reference: org.godotengine.godot.Godot but expected Reference: android.app.Activity (declaration of 'com.gameanalytics.godotgameanalytics.GodotGameAnalytics' appears in /data/app/com.gameanalytics.godottest-mtZ_tK2kr4ox-nPt89PH7A==/base.apk)
```

This can be reproduced using the example project and Godot 3.2.4-rc2. I think the fix is related to [this comment](https://github.com/Shin-NiL/Godot-Android-Admob-Plugin/issues/98#issuecomment-720642466).

It would be great if you could update the plugin source code and provide a compatible 3.2.4 plugin build on the download page. Thank you!

**Comment by the1schwartz** [comment]
Once 3.2.4 is out of release candidate we will make an update

**Comment by metinUr** [comment]
Godot 3.3 stable is released now, can you update the plugin?

**Comment by Mr-1311** [comment]
Godot 3.3 stable is released now, and I'm getting the same error:

 ```
 java.lang.VerifyError: Verifier rejected class com.gameanalytics.godotgameanalytics.GodotGameAnalytics: void com.gameanalytics.godotgameanalytics.GodotGameAnalytics.<init>(org.godotengine.godot.Godot) failed to verify: void com.gameanalytics.godotgameanalytics.GodotGameAnalytics.<init>(org.godotengine.godot.Godot): [0x6] register v2 has type Reference: org.godotengine.godot.Godot but expected Reference: android.app.Activity (declaration of 'com.gameanalytics.godotgameanalytics.GodotGameAnalytics' appears in /data/app/~~ipgrBL3J5xtpPKkoOyqrww==/com.the1311.astrorbit-i6BkxOMgFSFYIiw1fCsGfw==/base.apk!classes2.dex)
```
AdMob plugin updated their plugin to 3.3 on this [commit](https://github.com/Shin-NiL/Godot-Android-Admob-Plugin/commit/6e98b23e8e59c59b0b919bff85ce15286cfe452e#). Can you update this plugin too please?
Thanks.

**Comment by the1schwartz** [comment]
This has been fixed in the latest version now (v1.2.8)
